### DISCUSSION TITLE
is array type supported by tree editor?

### DISCUSSION THREAD

**Discussion by metadatadriven** [eclipse-emfcloud/emfcloud#227] [Theia Tree Editor]
Are arrays supported by the tree editor? 

I'm struggling to get the following example working as a tree node and wanted to check if arrays are supported before I dive into detail.

https://jsonforms.io/examples/array

Thanks!

**Comment by eneufeld** [comment]
Hi @metadatadriven ,

I'm unsure about the question.
The tree on the left hand side is custom and renders from an array of objects.
The details part that shows the selected data is based on jsonforms and should support all features that jsonforms offers. This would include the support for arrays.

If anything is missing or not working out as you expect, please let us know.

Best,
Eugen
